packages/manager/src/features/CloudPulse/Alerts/CreateAlert/GeneralInformation/ServiceTypeSelect.tsx

```diff
@@ -45,18 +45,24 @@ export const CloudPulseServiceSelect = (
     isLoading: serviceTypesLoading,
   } = useCloudPulseServiceTypes(true);
   const { control } = useFormContext<CreateAlertDefinitionForm>();
-  const { aclpBetaServices } = useFlags();
+  const { aclpServices } = useFlags();
   const getServicesList = React.useMemo((): Item<
     string,
     CloudPulseServiceType
   >[] => {
+    // If aclpServices is undefined, return all service types, else return the service types that are enabled and are present in the flag
     return serviceOptions?.data?.length
-      ? serviceOptions.data.map((service) => ({
-          label: service.label,
-          value: service.service_type as CloudPulseServiceType,
-        }))
+      ? serviceOptions.data
+          .filter(
+            (service) =>
+              aclpServices?.[service.service_type]?.alerts?.enabled ?? false
```

Comment:
```suggestion
              aclpServices?.[service.service_type]?.alerts?.enabled
```

same here, no fall back needed